src/components/NFTPortfolio/Empty.vue

```diff
@@ -13,26 +13,63 @@
         <div
           class="w-[42px] h-[42px] rounded-[50%] bg-shade-gray border-[2px] border-white"
         />
-        <Label
-          class="text-medium-gray mt-[12px]"
-          :text="
-            preset === 'collected'
-              ? $t('portfolio_collected_tab_no_item')
-              : $t('portfolio_created_tab_no_item')
-          "
-        />
+        <Label class="text-medium-gray mt-[12px]" :text="cardText" />
+        <i18n
+          v-if="shouldShowCrisp"
+          class="flex items-center text-medium-gray text-[14px] mt-[12px]"
+          path="nft_claim_claimed_empty_collected_help"
+        >
+          <button
+            place="help"
+            class="cursor-pointer text-like-green font-600 mx-[4px] hover:underline"
+            @click="handleClickHelp"
+          >
+            {{ $t('nft_claim_claimed_help') }}
+          </button>
+        </i18n>
       </div>
     </div>
   </NFTPortfolioCard>
 </template>
+
 <script>
+import crispMixin from '~/mixins/crisp';
+import { logTrackerEvent } from '~/util/EventLogger';
+
 export default {
+  name: 'NFTPortfolioEmpty',
+  mixins: [crispMixin],
   props: {
     // The preset of empty card, option: collected and created
     preset: {
       type: String,
       default: 'collected',
     },
   },
+  computed: {
+    cardText() {
+      return this.preset === 'collected'
+        ? this.$t('portfolio_collected_tab_no_item')
+        : this.$t('portfolio_created_tab_no_item');
+    },
+    shouldShowCrisp() {
+      return (
+        this.$route.name.includes('bookshelf') && this.preset === 'collected'
+      );
+    },
+  },
+  methods: {
+    handleClickHelp() {
+      logTrackerEvent(this, 'bookshelf', 'bookshelf_crisp_click', '', 1);
+      const res = this.openCrisp(
+        this.$t('nft_claim_claimed_empty_crisp_prefilled_message')
+      );
+      if (!res) {
+        window.open(
+          'https://discord.com/channels/763001015712350231/814761730349596712'
```

Comment:
Maybe use `https://go.crisp.chat/chat/embed/?website_id=....` , like in about nft book page `publishStoryURL`

Reply:
[✨ Add crisp chat site link as fallback](https://github.com/likecoin/liker-land/pull/1787/commits/3461fc6ebb231c63d7701517085dbb99086fb0b1)